core/kotlinx-coroutines-debug/src/CoroutineState.kt

```diff
@@ -23,13 +23,13 @@ public data class CoroutineState internal constructor(
 
     /**
      * [Job] associated with a current coroutine or [IllegalStateException] otherwise.
-     * May be later used in [DebugProbes.printHierarchy]
+     * May be later used in [DebugProbes.printJob]
      */
     public val job: Job get() = continuation.context[Job] ?: error("Continuation $continuation does not have a job")
 
     /**
      * [Job] associated with a current coroutine or null.
-     * May be later used in [DebugProbes.printHierarchy]
+     * May be later used in [DebugProbes.printJob]
      */
     public val jobOrNull: Job? get() = continuation.context[Job]
```

Comment:
This should have been mentioned in original debug review, but now after this change in docs the problem seems clear: There is an obvious copy & paste between `job` and `jobOrNull` properties. Moreover, having two properties like that is not idiomatic. Suggest: leave only property named `job: Job?` (nullable type). This would better reflect internal state of `CoroutineState` to the reader (the fact that is optionally has a `Job`). Anyone who need non-nullable job can always do `state.job!!` (no code will be affected now).